gammapy/irf/tests/test_energy_dispersion.py

```diff
@@ -48,3 +62,12 @@ def test_EnergyDispersion2D():
     e_val = (energy + energy2) / 2
     actual = edisp.evaluate(offset, e_val, migra)
     # assert_equal(lower > actual and actual > upper, True)
+
+    # Check RMF exporter
+    offset = Angle([0.612], 'deg')    
+    e_reco = EnergyBounds.equal_log_spacing(1,10,5,'TeV')
+    rmf = edisp.to_energy_dispersion(e_reco, offset)
+
+    actual = rmf._pdf_matrix[5]
```

Comment:
Ideally tests shouldn't have to access private things like `_pdf_matrix` here.
Maybe it makes sense to expose this as a property or make it a public member (i.e. remove the underscore)?